src/Constants.js

```diff
@@ -1,32 +1,46 @@
 const prod = {
```

Comment:
This file is getting ridiculously large and unmaintainable. In the next commit, just use one url each for dev and prod then just concatenate other parts of the api url when needed